apps/ledger-live-mobile/src/screens/MyLedgerDevice/index.tsx

```diff
@@ -46,7 +46,8 @@ const Manager = ({ navigation, route }: NavigationProps) => {
     tab = "CATALOG",
   } = route.params;
 
-  const { deviceId, deviceName, modelId } = device;
+  const { deviceId, modelId } = device;
+  const { deviceName } = result;
```

Comment:
this `deviceName` is the result of the `getDeviceName` command (in the list apps task), which will work in all cases (BLE but also USB)